package.json

```diff
@@ -22,7 +22,8 @@
   "workspaces": [
     "packages/*",
     "packages/utils/*",
-    "packages/plugins/*"
+    "packages/plugins/*",
+    "website/live-demo"
```

Comment:
I think we can just remove it. It can be independent from lerna and only bound to the website